test/test_intensify/test_hyperband.py

```diff
@@ -50,87 +51,89 @@ def setUp(self):
 
         self.logger = logging.getLogger(self.__module__ + "." + self.__class__.__name__)
 
-    @attr('slow')
-    def test_intensify_1(self):
+    def test_update_stage(self):
         """
-           test intensify for incumbent
+            test initialization of all parameters and tracking variables
         """
-
-        def target(x):
-            return (x['a'] + 1) / 1000.
-
-        taf = ExecuteTAFuncDict(ta=target, stats=self.stats)
-        taf.runhistory = self.rh
-
         intensifier = Hyperband(
-            tae_runner=taf, stats=self.stats,
-            traj_logger=TrajLogger(output_dir=None, stats=self.stats),
+            tae_runner=None, stats=self.stats, traj_logger=None,
             rng=np.random.RandomState(12345), deterministic=True, run_obj_time=False,
             instances=[1], initial_budget=0.1, max_budget=1, eta=2)
+        intensifier._update_stage()
+
+        self.assertEqual(intensifier.s, 3.0)
+        self.assertEqual(intensifier.s_max, 3.0)
+        self.assertEqual(intensifier.hb_iters, 0)
+        self.assertIsInstance(intensifier.sh_intensifier, SuccessiveHalving)
+        self.assertEqual(intensifier.sh_intensifier.initial_budget, 0.125)
+        self.assertEqual(intensifier.sh_intensifier.n_configs_in_stage.tolist(), [8.0, 4.0, 2.0, 1.0])
 
-        self.rh.add(config=self.config1, cost=1, time=1,
-                    status=StatusType.SUCCESS, instance_id=1,
-                    seed=0,
-                    additional_info=None)
+        # next HB stage
+        intensifier._update_stage()
 
-        inc, _ = intensifier.intensify(challengers=[self.config2],
-                                       incumbent=self.config1,
-                                       run_history=self.rh,
-                                       aggregate_func=average_cost)
+        self.assertEqual(intensifier.s, 2.0)
+        self.assertEqual(intensifier.hb_iters, 0)
+        self.assertEqual(intensifier.sh_intensifier.initial_budget, 0.25)
+        self.assertEqual(intensifier.sh_intensifier.n_configs_in_stage.tolist(), [4.0, 2.0, 1.0])
 
-        self.assertEqual(intensifier.s_max, 3)
-        self.assertEqual(inc, self.config1)
+        # HB iteration completed
+        intensifier.s = 0.0
```

Comment:
Could you please call update stage multiple times instead of taking this shortcut?